.travis.yml

```diff
@@ -1,31 +1,63 @@
 language: ruby
+
 sudo: false
 dist: trusty
+
 rvm:
   - 2.3.0
+
+# Allow Emacs snapshot builds to fail and don't wait for it
 matrix:
   fast_finish: true
   allow_failures:
     - env: EMACS_VERSION=snapshot
+
+# Cache stable Emacs binaries and packages
 cache:
-  - bundler
-  - apt: true
-  - directories:
+  bundler: true
+  apt: true
+  timeout: 691200
```

Comment:
Ah okay, so that timeout *is* 3 minutes by default, but it's not the time to wait before *clearing* the cache, it's the maximum time to wait for *building the cache*.  I was confused.

I don't think we need to change the default value then.